.github/workflows/build-and-release.yml

```diff
@@ -0,0 +1,44 @@
+name: Build and release
+
+'on':
+  push:
+    branches:
+      - maintenance/0.2.18
+  pull_request:
+    branches:
+      - maintenance/0.2.18
+jobs:
+  Test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          python-version: 2.7
+          fetch-depth: 0
```

Comment:
I added this step because otherwise there's an error that `setup.py` doesn't exist:
`ERROR: File "setup.py" not found. Directory cannot be installed in editable mode: /home/runner/work/refextract/refextract`

Reply:
I didn't mean the step, I meant L17, `python-version` is not used by the checkout action. BTW I also didn't mean a separate PR, just a separate commit in the same PR would have be ok.